docs/source/install/index.rst

```diff
@@ -3,15 +3,15 @@ Installation
 
 For a quick evaluation of |st2| and package-based installation, start from
 :doc:`Overview <./overview>`, and proceed with the guide for your Linux distribution.
-
+Please note that only 64 bit architecture is currently supported.
 
 .. rubric:: Installations
 
 .. toctree::
     :maxdepth: 1
 
     Reference Deployment <overview>
-    Ubuntu / Debian <deb>
+    Ubuntu / Debian 14.04 (Trusty) <deb>
```

Comment:
>there are debian wheezy and jessie packages in unstable repo

Here's a fun problem. I basically don't want to claim support unless it goes through our CI machinery. If you are on #stackstorm, you'd see atrocious numbers of CI failures just for 3 OS for which we CI regularly. Most of the times, it is a 3rd party thing we depend on - GNU TLS package cloud, AWS APIs timing out etc. There is a huge pain now to make sure 3 OSes get CIed correctly. If we add more to the mix, we'll essentially kill ourselves with just debugging build failures. So I am extremely skeptical about adding more OSes. From a customers' point, no one has asked for those. Many enterprises are switching to ubuntu server images if not using them already.